cmd/website/macros/macros.go

```diff
@@ -66,7 +66,21 @@ func expandCf(line string) string {
 		} else {
 			sep = ", "
 		}
-		fmt.Fprintf(&buf, "%s[`%s`](#%s)", sep, target, target)
+		if i := strings.IndexRune(target, ':'); i == -1 {
+			// This handles the @cf uses in the `builtin:` namespace to other
+			// commands in the same namespace.
+			fmt.Fprintf(&buf, "%s[`%s`](#%s)", sep, target, target)
+		} else {
+			module := target[:i]
+			href := target[i+1:]
```

Comment:
This variable is not the href, but the unqualified symbol name; call it `symbol`.

Reply:
Oops, yes, agreed.